pkg/process/config/yaml_config.go

```diff
@@ -384,7 +385,8 @@ func (a *AgentConfig) LoadProcessYamlConfig(path string) error {
 	if config.Datadog.GetBool("orchestrator_explorer.enabled") {
 		a.OrchestrationCollectionEnabled = true
 		// Set clustername
-		if clusterName := clustername.GetClusterName(); clusterName != "" {
+		hostname, _ := coreutil.GetHostname()
```

Comment:
Is it safe to ignore every where the possible error returned by `coreutil.GetHostname()` 
If yes why not changing the function signature to only return a string?

Reply:
From what I looked up in the code the agents use the error to exit at startup:
https://github.com/DataDog/datadog-agent/blob/dc1d3307f695544c394728ae56293c1d387690c8/cmd/agent/app/run.go#L223-L226
https://github.com/DataDog/datadog-agent/blob/dc1d3307f695544c394728ae56293c1d387690c8/cmd/cluster-agent/app/app.go#L162-L164
so we should definitely keep the error. In my use cases I think we can assume that the hostname is already cached otherwise the agent would have stopped already.